Adding verifiedPublisher to Application and ServicePrincipal (#9827)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/verifiedpublisher.md

---
title: "verifiedPublisher resource type"

> description: "Verified publisher of the application."

Reviewer: ```suggestion
description: "Represents the verified publisher of the application."
```

localization_priority: Normal
doc_type: resourcePageType
ms.prod: "microsoft-identity-platform"
author: "jesakowi"
---

# verifiedPublisher resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Verified publisher of an [application](application.md). For more information, see [Publisher Verification Overview](https://docs.microsoft.com/azure/active-directory/develop/publisher-verification-overview). All properties are set as a result of [Set Verified Publisher](../api/application-setverifiedpublisher.md) and can only be removed with [Unset Verified Publisher](../api/application-unsetverifiedpublisher.md).

Reviewer: ```suggestion
Represents a verified publisher of an [application](application.md). For more information, see [Publisher Verification Overview](https://docs.microsoft.com/azure/active-directory/develop/publisher-verification-overview). Verified publishers are set using [Set Verified Publisher](../api/application-setverifiedpublisher.md) and can only be removed using [Unset Verified Publisher](../api/application-unsetverifiedpublisher.md).
```

## Properties

| Property | Type | Description |
|:---------------|:--------|:----------|

> |displayName|String|The verified publisher name from the app publisher's Partner Center account.|

Reviewer: Is the user going to understand "app publisher's Partner Center account"?

Author: Good call- fixed to be more clear
